README.md

```diff
@@ -1,29 +1,127 @@
 # Quarkus Test Suite
 
+The aim of this test suite is to validate Quarkus framework features along the time. Under the hood we are using [Quarkus-test-framework](https://github.com/quarkus-qe/quarkus-test-framework) that we designed for this proposal. 
+
 The test suite includes:
 - Bare metal scenario testing
 - OpenShift scenario testing
+- Native testing
+
+## How-to run Quarkus test suite
+
+Please before start be sure that you have all [prerequisites](#Prerequisites) installed.
```

Comment:
This is not good for the good first impression

We can assume bare metal use case as the first option, so I would cover `mvn clean verify`

next to it cover how to run native mode in bare metal - `mvn clean verify -Dnative` ?

third item would be how to run concrete test in jvm mode in baremetal

after that you can add line like "Refer to the section below for additional detail"

Reply:
The impression needs to be "Nice, it's easy to use"

Reply:
I've re-worked this section. Let me know how do you see now!